test/index.js

```diff
@@ -724,11 +706,9 @@ test ('equals', () => {
   eq (Z.equals (Math.sin, Math.cos), false);
   eq (Z.equals (Identity (Identity (Identity (0))), Identity (Identity (Identity (0)))), true);
   eq (Z.equals (Identity (Identity (Identity (0))), Identity (Identity (Identity (1)))), false);
-  eq (Z.equals (Useless, Useless), true);
```

Comment:
I removed this. But prior to https://github.com/sanctuary-js/sanctuary-type-classes/pull/154/files#diff-5bb8db779819ddef5956a5d9d5949c05ef7445237656ca37bf2f02720271440bL676-R703, you had it in the test suite to explicitly test that Useless and Useless are not equal. It seems that you were testing that garbage input _actually_ produce garbage output. Should we have tests like that? I think we can leave unspecified behaviour totally unspecified.